### PR TITLE
Oe 519/pw clever login

### PR DESCRIPTION
**Pull request by ayan1229**

### CODE REVIEW

File: playwright/support/utils.ts [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
   pageId: keyof typeof pages
 ): boolean => {
   return pages[pageId].regex.test(currentPath);
 };
+
+export const currentDomainMatchesDomainId = (
+  currentDomain: string,
+  externalPageId: keyof typeof externalPages
+): boolean => {
+  return externalPages[externalPageId].domain.includes(currentDomain);
```
[the remark this answers is not in the document]
Comment (samanthaandrews) [reply playwright/support/utils.ts:14]: i am so wrong! there is a `string.includes()` method that i completely forgot about.

File: playwright/features/cleverSignIn.feature [hunk 2]
```diff
@@ -10,5 +10,12 @@ Feature: As a user I want to sign into Open Ebooks Web by using clever credentia
      And I fill in the "clever username textfield" with "OPENEBOOKS_CLEVER_HIGH_SCHOOL_USER_USERNAME"
      And I fill in the "clever password textfield" with "OPENEBOOKS_CLEVER_HIGH_SCHOOL_USER_PW"
      And I click the "clever login button"
+     And I am directed to the "sign in clever callback" page
+     And I am directed to the "open ebook circulation" page
+     And I click the "sign out button"
+     And I click the "confirm sign out button"
      And I am on the "home" page
```
Comment (samanthaandrews) [inline playwright/features/cleverSignIn.feature:17]: I'm realizing the naming of these steps is a little confusing (my fault for naming them this way originally!). Let's change these steps from `I am on the ${name} page` to something that signifies an action. Something like `I visit the ${name} page` or `I go to the ${name} page`.

**Review by samanthaandrews — APPROVED** [review]
This is such great work!

**Comment by ayan1229** [comment]
Thanks @samanthaandrews ! I am going to merge it now

**Pull request merged**